README.md

```diff
@@ -233,7 +238,7 @@ For example, in VS Code, click `Debug` > `Add Configuration...` > `Java: Attach`
 
 6. When you are done, press Ctrl-C in the console to terminate dev mode and stop your server.
 
-## VS Code extension
+## VS Code extension (Currently maven only)
 
 The [Liberty Dev](https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext) extension is an IDE specific extension for VS Code that lets you interact with dev mode through menu options as an alternative to using the console.
```

Comment:
Change `Liberty Dev` to `Open Liberty Tools` since it has been renamed. Also in step 2 and the title text in step 6 below.  Step 5 is fine as-is though.